### DISCUSSION TITLE
Creating meta files to a added textures

### DISCUSSION THREAD

**Discussion by Dr4gma** [space-wizards/space-station-14#16055] [Technical]
Hi all, do I have to add my own meta file when adding a new hat texture for example? this question might have an obvious answer but I need to know. Thanks.

**Comment by deltanedas** [comment]
you only have 1 meta file per rsi so if its significant make a separate rsi
if its just changing a single texture or something you could probably just append that to copyright string, though im no expert for that

**Reply by Dr4gma** [reply]
Thanks for the answer. It's a seperate .rsi
